jython/DCC-EX/SendDCCppMessages.py

```diff
@@ -44,3 +44,7 @@ def send(strMessage) :
 send("S")               # request list of Sensor definitions
 send("Z")               # request list of Output definitions
 send("s")               # request update of all state lists
+# example TrackManager messages, these are from DCC-EX 5.0+
+send("= B PROG")   # Set track B to PROG
+send("= B MAIN")   # Set track B back to MAIN
+send("=")          # request updated trackmanager status
```

Comment:
This file has no empty line at the end of the file.

Reply:
> This file has no empty line at the end of the file.

Will that make a difference? I can add it back.

Reply:
You can leave it as is. But for other files, like Java files, there is a requirement that the file ends with an empty line.